integration/terraform/ec2/linux/variables.tf

```diff
@@ -0,0 +1,72 @@
+variable "ec2_instance_type" {
+  type = string
+  default = "t3a.xlarge"
+}
+
+variable "key_name" {
+  type = string
+  default = "cwagent-integ-test-key"
+}
+
+variable "iam_instance_profile" {
+  type = string
+  default = "CloudWatchAgentServerRole"
+}
+
+variable "vpc_security_group_ids" {
+  type = list(string)
+  default = ["sg-013585129c1f92bf0"]
```

Comment:
This smells fishy. This is a specific security group in an individual AWS account, right?

Reply:
Don't think it's necessary to make it a secret.